en/docs/develop/restapis/userstore.yaml

```diff
@@ -592,38 +577,31 @@ components:
         name:
           type: string
           example: JDBC-SECONDARY
-          description: Domain name of the secondary user store
+          description: domain name of the secondary user store
```

Comment:
Shouldn't this start with a capital letter? There is an inconsistency when compared with other similar descriptions